blog/__init__.py

```diff
@@ -0,0 +1 @@
+"""Configure files for blog."""
```

Comment:
It should not be in imperative mood: just importing this package won't emit any actions. This is a description, no a command.